tests/store/db/test_utils.py

```diff
@@ -21,9 +23,23 @@ def test_create_sqlalchemy_engine_inject_pool_options():
                 "mydb://host:port/",
                 pool_pre_ping=True,
                 pool_size=2,
-                pool_recycle=3600,
                 max_overflow=4,
+                pool_recycle=3600,
                 echo=True,
+                poolclass=NullPool,
+            )
+
+
+def test_create_sqlalchemy_engine_not_inject_pool_options():
```

Comment:
Looks like this test is equivalent to `test_create_sqlalchemy_engine_no_pool_options`.

Reply:
Thanks. Removed.